Remotion/Web/Development.WebTesting/WebDriver/Configuration/Chromium/ExtendedChromiumOptions.cs

```diff
@@ -14,15 +14,16 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with re-motion; if not, see http://www.gnu.org/licenses.
 // 
+using System;
 using JetBrains.Annotations;
 using OpenQA.Selenium.Chrome;
 
-namespace Remotion.Web.Development.WebTesting.WebDriver.Configuration.Chrome
+namespace Remotion.Web.Development.WebTesting.WebDriver.Configuration.Chromium
 {
   /// <summary>
   /// Extends <see cref="ChromeOptions"/> provided by Selenium with additional data used by the web testing infrastructure.
   /// </summary>
-  public class ExtendedChromeOptions : ChromeOptions
+  public class ExtendedChromiumOptions : ChromeOptions
```

Comment:
How can we derive ExtendedChromIUMOptions from ChromEOptions?

How does the EdgeDriver work? Does it have EdgeOptions? Does it reuse ChromeOptions?

Reply:
The correct `EdgeDriverService` will come with Selenium 4. We should use the chrome one until our upgrade.

Reply:
okay, please document the current and future inheritance hierarchies in a follow-up jira task so we can compare/verifiy that the future model will make sense and that the current model can be adapted to the future model without breaking changes.

Reply:
Good. Then we need "ExtendedChromeOptions" and "ExtendedEdgeOptions" instead of one shared type and possibly an interface IExtendedChromiumOptions if we actually need to reference the members of the ExtendedOptions from within shared code.

Reply:
Added (RM-7275 - Upgrade Selenium to 4.0.0)[https://re-motion.atlassian.net/browse/RM-7275].